Project.toml

```diff
@@ -49,7 +49,7 @@ WriteVTK = "64499a7a-5c06-52f2-abe2-ccb03c286192"
 [compat]
 Adapt = "2.0.2"
 ArgParse = "1.1"
-CLIMAParameters = "0.1.6"
+CLIMAParameters = "= 0.1.8"
```

Comment:
Is this what we want? @jakebolewski

Reply:
In package manager I tried simply `up CLIMAParameters` but was constantly getting package errors, even after I fixed my Julia version. The only thing that worked for me was to `rm CLIMAParameters` and then `add CLIMAParameters`. I don't know if thats the result of it. 

I can manually edit this file and remove the exact version. I don't think I can achieve the same inside the package manager.

Reply:
Ah, that should be fine then. Thanks!

Reply:
I think the min version is probably better, even if for now defacto there is an assumed pinned version for ClimateMachine (another project that depends on ClimaParameters could add some parameters that are not needed by ClimateMachine).